ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -323,6 +323,15 @@ VSC:
   - speed_frequency_sensor
   - power_sensor
   - voltage_sensor
+  - phase1_phase2_line_voltage_sensor
```

Comment:
the phase-level fields are too vague to have on this abstract type - are your fields input or output voltages/currents?

Reply:
Removed the phase-level fields as agreed with you , included voltage, current sensor in CHDXVSC.